build.gradle

```diff
@@ -77,7 +77,7 @@ dependencies {
 	implementation group: 'bio.terra.cloud-resource-lib', name: 'google-bigquery', version: '0.4.0'
 	implementation group: 'bio.terra.cloud-resource-lib', name: 'google-cloudresourcemanager', version: '0.4.0'
 	implementation group: 'bio.terra.cloud-resource-lib', name: 'google-serviceusage', version: '0.4.0'
-	implementation group: 'bio.terra.cloud-resource-lib', name: 'google-storage', version: '0.4.0'
+	implementation group: 'bio.terra.cloud-resource-lib', name: 'google-storage', version: '0.5.0'
```

Comment:
bumped to 0.7.0 since that's out now.
in general, I prefer only changing dependencies relevant to PRs for the sake of (relatively) smaller and more focused changes. separately, it's probably worth a pass updating our dependencies semi-frequently